scripts/mock_control_api.py

```diff
@@ -71,8 +71,7 @@ def get_results(
         updated_at = decoded["updated_at"]
 
         # this is the org id or project key id depending on the result type
-        entity_key = "org_id" if entity == EntityType.ORG else "project_key"
-        entity_id = decoded[entity_key]
+        entity_id = decoded["id"]
```

Comment:
since synapse needs to parse the cursors now, it's better to make this just "id" - agnostic of org or projectkey or whatever the identifier is